modules/otlp/otlp-to-lgtm/README.md

```diff
@@ -0,0 +1,52 @@
+# OTLP to LGTM Stack Module
+
+Module to ingest OTLP data and then send it to Loki, Mimir and Tempo stacks locally or in GrafanaCloud.
+
+
+## Module arguments
+
+The following arguments are supported when passing arguments to the module
+loader:
+
+| Name | Type | Description | Default | Required
+| ---- | ---- | ----------- | ------- | --------
+| `otlp_http_endpoint` | `string` | The OTLP HTTP server URL. | `"0.0.0.0:4318"` | no
+| `otlp_grpc_endpoint` | `string` | The OTLP gRPC server URL. | `"0.0.0.0:4317"` | no
```

Comment:
should we remove these or implement their usage in the module

Reply:
Testing them now along with some other doc changes.